crates/radix-engine-toolkit-uniffi/Cargo.toml

```diff
@@ -18,18 +18,18 @@ radix-engine-interface = { workspace = true }
 radix-engine-toolkit = { path = "../radix-engine-toolkit" }
 
 # The UniFFI crate for generating bindings to other languages
-uniffi = { git = "https://github.com/mozilla/uniffi-rs/", tag = "v0.25.2", features = ["cli"] }
+uniffi = { git = "https://github.com/mstrug-rdx/uniffi-rs/", branch = "bugfix/result-type-in-lower-return", features = ["cli"] }
```

Comment:
Can we switch to a `rev` or a `tag` everywhere?

Reply:
done